bgpd/bgpd.c

```diff
@@ -2577,6 +2582,10 @@ int peer_group_delete(struct peer_group *group)
 
 	for (ALL_LIST_ELEMENTS(group->peer, node, nnode, peer)) {
 		other = peer->doppelganger;
+
+		if (CHECK_FLAG(peer->flags, PEER_FLAG_CAPABILITY_ENHE))
```

Comment:
Not necessary in this case, since the peers are deleted when the peer-group is deleted.  This fix just turns off the RAs prior to the peers getting removed.